boards/arm/nrf52810_pca10040/Kconfig

```diff
@@ -6,4 +6,9 @@
 
 if BOARD_NRF52810_PCA10040
 
+config BOARD_HAS_DCDC
```

Comment:
Kconfigs with _HAS_  should not be prompted, those should be in general hidden Kconfigs.  Also, where are those being used? You might want to change the name of the Kconfig and the prompt text .